autogpt_platform/frontend/src/components/monitor/FlowRunInfo.tsx

```diff
@@ -1,79 +1,160 @@
-import React, { useCallback } from "react";
-import AutoGPTServerAPI, { GraphMeta } from "@/lib/autogpt-server-api";
+import React, { useCallback, useEffect, useMemo, useState } from "react";
+import AutoGPTServerAPI, {
+  BlockIORootSchema,
+  Graph,
+  GraphMeta,
+  NodeExecutionResult,
+} from "@/lib/autogpt-server-api";
 import { FlowRun } from "@/lib/types";
 import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card";
 import Link from "next/link";
 import { Button, buttonVariants } from "@/components/ui/button";
 import { IconSquare } from "@/components/ui/icons";
-import { Pencil2Icon } from "@radix-ui/react-icons";
+import { ExitIcon, Pencil2Icon } from "@radix-ui/react-icons";
 import moment from "moment/moment";
 import { FlowRunStatusBadge } from "@/components/monitor/FlowRunStatusBadge";
+import RunnerOutputUI, { BlockOutput } from "../runner-ui/RunnerOutputUI";
 
 export const FlowRunInfo: React.FC<
   React.HTMLAttributes<HTMLDivElement> & {
     flow: GraphMeta;
     flowRun: FlowRun;
   }
 > = ({ flow, flowRun, ...props }) => {
+  const [isOutputOpen, setIsOutputOpen] = useState(false);
+  const [graph, setGraph] = useState<Graph | null>(null);
+  const [executionResults, setExecutionResults] = useState<
+    NodeExecutionResult[] | null
+  >(null);
+  const [blockOutputs, setBlockOutputs] = useState<BlockOutput[]>([]);
+  const api = useMemo(() => new AutoGPTServerAPI(), []);
+
+  // Fetch graph and execution data
+  useEffect(() => {
+    api.getGraph(flow.id, flow.version).then((graph) => {
+      setGraph(graph);
+    });
+    api.getGraphExecutionInfo(flow.id, flowRun.id).then((executions) => {
+      setExecutionResults(executions);
+    });
```

Comment:
I had to get executions info because `flowRun.nodeExecutionResults` is always `undefined`

Reply:
I think there are `with_runs` query params being added on /graphs ? can we utilize that?
Also, are we going to fetch literally all the executions of the graph?

Reply:
Can we fetch the output only when it's clicked ?

Reply:
> I think there are with_runs query params being added on /graphs ? can we utilize that?

Why? It doesn't return actual results but only all runs metadata.

> Also, are we going to fetch literally all the executions of the graph?

No, it fetches execution info for a single run that is selected. I'm unsure about fetching all runs metadata on the Monitor but that's unrelated to this PR.

> Can we fetch the output only when it's clicked ?

Done